src/main/webapp/content/files/companionDiagnosticDevices/companion_diagnostic_devices.json

```diff
@@ -0,0 +1,16753 @@
+[
```

Comment:
Same question as cancer types

Reply:
We don't have an API client that connects to oncokb-curation (which is where the CDx data is). We are using a static json file for now.